src/CommandHandler.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Exan\Fenrir;
+
+use Exan\Fenrir\Command\FiredCommand;
+use Exan\Fenrir\Const\Events;
+use Exan\Fenrir\Parts\ApplicationCommand;
+use Exan\Fenrir\Rest\Helpers\Command\CommandBuilder;
+use Exan\Fenrir\Websocket\Events\InteractionCreate;
+use Exan\Fenrir\Websocket\Events\Ready;
+
+class CommandHandler
+{
+    private bool $activated = false;
+
+    private array $commands = [];
+
+    private bool $devMode = false;
+
+    public function __construct(private Discord $discord, private ?string $devGuildId = null)
+    {
+        if (!is_null($this->devGuildId)) {
+            $this->devMode = true;
+        }
```

Comment:
```suggestion
        $this->devMode = ($this->devGuildId !== null) ? true : false;
```
Comparing to null is faster than using `is_null` as there's no function call overhead

Reply:
I find `is_null` easier to read, micro-optimizing isn't worth it imo

Reply:
That's fine, thought I'd run it by you.